### PR TITLE
Created a method that extracts data as list of dictionaries from data validator.

### PR DESCRIPTION
**Pull request by bissella**
Created a method that extracts data as list of dictionaries from data validator.

### CODE REVIEW

File: data/test.csv [hunk 1]
```diff
@@ -1 +1 @@
-column 1,column 2,column 31,2,3
\ No newline at end of file
+column 1,column 2,column 31,2,33,4,5
```
Comment (jTaylo99) [inline data/test.csv]: Leftover changes from testing csv

File: src/data_holder.py [hunk 2]
```diff
@@ -43,6 +46,13 @@ def validate_row(self, row: Data):
         if list(row.__dict__.keys()) != self._columns:
             raise TypeError
 
+    def all_data_to_dict(self):
```
Comment (jTaylo99) [inline src/data_holder.py:49]: Docstring would be good here

**Review by jTaylo99 — CHANGES_REQUESTED** [review]
Only minor changes needed.

**Review by jTaylo99 — APPROVED** [review]
Looks good

**Comment by bissella** [comment]
Made minor changes.

**Pull request merged**